File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profISEEInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profISEEInfo extends Auditable {
    public  ISEEInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }

    //public int? ISEEClassTypeId { get; set; }
    //    public virtual ISEEClassType ISEEClassType { get; set; }

    public iseeClassTypeName: string

    public iseeClassTypeId: number;

    public iseeValue: number;

    public point: number;

    public iseeFamilyIncome: number;

    public ispAmount: number;

    public iseAmount: number;

    public isrAmount: number;

    public identificationNumber: string;

    public submittedDate: Date;

    public deliveryDate: Date;

    public expiryDate: Date;
        
}

File: Architecture.Core/Entities/ProfISEEInfo.cs
using System;
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;

namespace Architecture.Core.Entities
{
    public class ProfISEEInfo : Auditable
    {
        public int ISEEInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public int? ISEEClassTypeId { get; set; }
        public virtual ISEEClassType ISEEClassType { get; set; }
        public decimal ISEEValue { get; set; }
        public decimal Point { get; set; }
        public decimal ISEEFamilyIncome { get; set; }
        public decimal ISPAmount { get; set; }
        public decimal ISEAmount { get; set; }
        public decimal ISRAmount { get; set; }
        public string IdentificationNumber { get; set; }
        public DateTime? SubmittedDate { get; set; }
        public DateTime? DeliveryDate { get; set; }
        public DateTime? ExpiryDate { get; set; }
    }
}
